NEWS.md

```diff
@@ -1,5 +1,11 @@
 # rsconnect 0.8.30 (development version)
 
+* `deployApp()` is more aggressive about saving deployment data, which should
```

Comment:
aside: I totally missed that we have `forgetDeployment` to help clean-up those pesky repeats I'm always creating when testing!